packages/instrumentation-amqplib/package.json

```diff
@@ -0,0 +1,68 @@
+{
+    "name": "opentelemetry-instrumentation-amqplib",
+    "version": "0.2.2",
+    "description": "open telemetry instrumentation for the `amqplib` package",
+    "keywords": [
+        "amqplib",
+        "opentelemetry",
+        "rabbitmq",
+        "AMQP 0-9-1"
+    ],
+    "homepage": "https://github.com/aspecto-io/opentelemetry-amqplib",
+    "license": "Apache-2.0",
+    "main": "dist/src/index.js",
+    "files": [
+        "dist/src/**/*.js",
+        "dist/src/**/*.d.ts",
+        "LICENSE",
+        "README.md"
+    ],
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/aspecto-io/opentelemetry-amqplib.git"
+    },
+    "scripts": {
+        "build": "tsc",
+        "prepare": "yarn run build",
+        "test": "mocha",
+        "watch": "tsc -w",
+        "version:update": "node ../../scripts/version-update.js",
+        "version": "yarn run version:update",
+        "test-all-versions": "tav",
+        "test:ci": "yarn test-all-versions",
+        "test:docker:run": "docker run -d --hostname demo-amqplib-rabbit --name amqplib-unittests -p 22221:5672 rabbitmq:3"
+    },
+    "bugs": {
+        "url": "https://github.com/aspecto-io/opentelemetry-ext-js/issues"
+    },
+    "dependencies": {
+        "@opentelemetry/api": "^0.17.0",
+        "@opentelemetry/instrumentation": "^0.17.0",
+        "@opentelemetry/semantic-conventions": "^0.17.0",
+        "test-all-versions": "^5.0.1"
+    },
+    "devDependencies": {
+        "@opentelemetry/api": "^0.17.0",
+        "@opentelemetry/core": "^0.17.0",
+        "@opentelemetry/node": "^0.17.0",
+        "@opentelemetry/tracing": "^0.17.0",
+        "@types/amqplib": "^0.5.17",
+        "@types/lodash": "^4.14.168",
+        "@types/mocha": "^8.2.0",
+        "@types/sinon": "^9.0.11",
+        "amqplib": "^0.7.1",
+        "expect": "^26.6.2",
+        "lodash": "^4.17.21",
```

Comment:
you can instead install only `lodash.times`
https://www.npmjs.com/package/lodash.times

Reply:
Thanks, I wasn't aware this is possible. would prefer to use it from now on.
Specifically in this case I am using both `times` and `range` in tests, and this is dev dependency anyhow so it does not impact the end user.
It will also make it a bit more friendly for future maintenance in my opinion.
What do you think about keeping it this way?

Reply:
Can micro improve ci time but doesn't really matter